entities/OrderedStation.php

```diff
@@ -96,6 +96,16 @@ public function linkedListPathAsArray() {
     return array_map('intval', $r);
   }
 
+  /** Will take a coordinate and return the closest stationid
+   *
+   * Will take a coordinate and return the closest stationid
+   *
+   * @param float $coord Coordinates from client
+   * @return [OrderedStation] [NYI]
+   */
+  public function getStationFromCoord($coord){
```

Comment:
I'm suspicious about this one... It's going to get a station from a single float?

Should a "coord" be it's own data type? Should it be a tuple? Should the method accept a $lat, $lng instead? 

Also, once again, general good practice: if you're not implementing it leave a "TODO:" or something in the code to make it abundently clear that this function is intentionally broken. (Otherwise someone reading it might be like "WTF? Let me spend an hour pondering why he might have done this...")